README.md

```diff
@@ -0,0 +1,10 @@
+# ensembl-pipeline
+Job management for the Ensembl Genebuild pipeline
+
+**This repository is deprecated**, we use https://github.com/Ensembl/ensembl-hive for our pipelines
+
+All scripts which we still use have been moved to https://github.com/Ensembl/ensembl-ananlysis
+
+If you cannot find a script from this repository in the ensembl-analysis repository or for any request, please contact us
+* on the Ensembl developers list: dev@ensembl.org
+* through our helpdesk: helpdesk@ensembl.org 
```

Comment:
I thought the policy was not to show our email addresses in the repositories and instead link to http://www.ensembl.org/Help/Contact